src/Events.Functions/Constants/EventConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Altinn.Platform.Events.Functions.Constants
+{
+    /// <summary>
+    /// Shared constants for Events.Functions project
+    /// </summary>
+    public static class EventConstants
+    {
+        /// <summary>
+        /// The cloud event type for subscription validation
+        /// </summary>
+        public static string ValidationType => "platform.events.validatesubscription";
```

Comment:
I prefer simple const.
```suggestion
        public const string ValidationType = "platform.events.validatesubscription";
```